packages/fuels-abigen-macro/tests/test_projects/use_enum_input/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
```

Comment:
`Forc.lock` files shouldn't be committed! Add `Forc.lock` to this new project's `.gitignore`

Reply:
A shoot I wasn't sure, almost asked the question haha